js/script.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function daysInYear(year) {
+  try {
+    if (!Number.isInteger(year) || !Number(year) > 0){
+      throw new Error();
+    }
+    console.log(daysOnLeapYear(year) ? 366 : 365);
```

Comment:
This method should return number of days. It prints number of days into the console.